Containerfile

```diff
@@ -20,16 +7,23 @@ ENV PYTHONUNBUFFERED=1
 
 WORKDIR /app/
 
-# Copy built binary from builder
-COPY --from=builder /build/node_modules/yuglify/dist/* /app/node_modules/yuglify/dist/
-
 # Copy files to container
 COPY pyproject.toml README.md ./docker/entrypoint.sh /app/
 COPY ./bagitobjecttransfer /app/bagitobjecttransfer
 
-# Install Python dependencies
+# # Install Python dependencies
 RUN pip install .
 
+# Install Node.js
+RUN apt-get update && apt-get install -y curl && \
```

Comment:
Another comment which we can discuss - NodeJS was installed in the previous image, `FROM docker.io/node:20 AS builder`, so it shouldn't be required to install it in the Python image.

Reply:
Note: try swapping to this image that includes both Python and Node
https://hub.docker.com/r/nikolaik/python-nodejs
python3.10-nodejs22-slim

Reply:
Swapped to the above mentioned image.

Resolved by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/248/commits/5347e12e5a114de6abddc1df1870926f8fc52de2